tests/support/iframe-commands.js

```diff
@@ -1,83 +1,16 @@
-Cypress.Commands.add(
-  'getIframeBody',
-  (getIframeOpts = {}, index = 0, containerSelector = '.iframeContainer') => {
-    return cy
-      .get(`${containerSelector} iframe`, getIframeOpts)
-      .eq(index)
-      .wait(1000)
-      .iframe();
-  },
-);
+Cypress.Commands.add('getIframeBody', () => {
+  // get the iframe > document > body
+  // and retry until the body element is not empty
+  cy.log('getIframeBody');
 
-Cypress.Commands.add('iframe', { prevSubject: 'element' }, $iframe => {
-  Cypress.log({
-    name: 'iframe',
-    consoleProps() {
-      return {
-        iframe: $iframe,
-      };
-    },
-  });
-  return new Cypress.Promise(resolve => {
-    onIframeReady(
-      $iframe,
-      () => {
-        resolve($iframe.contents().find('body'));
-      },
-      () => {
-        $iframe.on('load', () => {
-          resolve($iframe.contents().find('body'));
-        });
-      },
-    );
-  });
+  return (
+    cy
+      .get('.iframeContainer iframe', { log: false })
```

Comment:
I think we should get the iframe which is currently visible. Luigi can store many iframes inside this container.
![image](https://user-images.githubusercontent.com/22825433/115687322-585ff580-a35a-11eb-9efe-d12bf9ce6e6b.png)